parquet_marshaller.go

```diff
@@ -368,14 +364,14 @@ func parquetMarshalCommitInfo(add *action.CommitInfo, obj interfaces.MarshalObje
 	return nil
 }
 
+/**
+ * commitInfo isn't included in a checkpoint parquet file:
```

Comment:
- use `//` for function documentation.
- start with function name - parquetUnmarshalCommitInfo